Makefile

```
@@ -69,116 +58,131 @@ handler:
 
-# Run go fmt against code
+## Run go fmt against code
 fmt:
 	go fmt ./...
 
-# Run go vet against code
+## Run go vet against code
 vet:
 	go vet ./...
 
-# Run golangci-lint against code
+## Run golangci-lint against code
 lint:
 	golangci-lint run --timeout 5m0s
 
-# Generate code
+## Generate code
 generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths="./..."
 
-colima-build-manager: manager
-	nerdctl build --namespace ${CONTAINERD_REGISTRY_PREFIX} --build-arg TARGETARCH=${OS_ARCH} -t ${MANAGER_IMAGE} -f bin/manager/Dockerfile ./bin/manager/
+# Lima actions
+## Create a new lima cluster and deploy the chaos-controller into it
+lima-all: lima-start lima-install-cert-manager lima-build-all lima-install
+
+## Rebuild the chaos-controller images, re-install the chart and restart the chaos-controller pods
+lima-redeploy: lima-build-all lima-install lima-restart
+
+## Install cert-manager chart
+lima-install-cert-manager:
+	$(KUBECTL) apply -f https://github.com/jetstack/cert-manager/releases/download/v1.9.1/cert-manager.yaml
+
+## Install CRDs and controller into a lima k3s cluster
+## In order to use already built images inside the containerd runtime
+## we override images for all of our components to the expected namespace
+lima-install: manifests
+	helm template \
+		--set controller.enableSafeguards=false \
+		--set controller.expiredDisruptionGCDelay=${EXPIRED_DISRUPTION_GC_DELAY} \
+		./chart | $(KUBECTL) apply -f -
 
-colima-build-injector: injector
-	nerdctl build --namespace ${CONTAINERD_REGISTRY_PREFIX} --build-arg TARGETARCH=${OS_ARCH} -t ${INJECTOR_IMAGE} -f bin/injector/Dockerfile ./bin/injector/
+## Uninstall CRDs and controller from a lima k3s cluster
+lima-uninstall:
+	helm template ./chart | $(KUBECTL) delete -f -
 
-colima-build-handler: handler
-	nerdctl build --namespace ${CONTAINERD_REGISTRY_PREFIX} --build-arg TARGETARCH=${OS_ARCH} -t ${HANDLER_IMAGE} -f bin/handler/Dockerfile ./bin/handler/
+## Restart the chaos-controller pod
+lima-restart:
+	$(KUBECTL) -n chaos-engineering rollout restart deployment chaos-controller
 
-colima-build: colima-build-manager colima-build-injector colima-build-handler
+## Build all images and import them in lima
+lima-build-all: lima-build-manager lima-build-injector lima-build-handler
 
-# Build the docker images
-# to ease minikube/colima base minikube docker image load, we are using tarball
-# we encountered issue while using docker+containerd on colima to load images from docker to containerd
-# this Make target are used by the CI
-minikube-build-manager: manager
+docker-build-manager: manager
 	docker build --build-arg TARGETARCH=${OS_ARCH} -t ${MANAGER_IMAGE} -f bin/manager/Dockerfile ./bin/manager/
-	docker save -o ./bin/manager/manager.tar ${MANAGER_IMAGE}
-	minikube image load --daemon=false --overwrite=true ./bin/manager/manager.tar
+	docker save ${MANAGER_IMAGE} -o ./bin/manager.tar.gz
```

You may want to put those files under each dedicated bins or add them to the `.gitignore`
```suggestion
	docker save ${MANAGER_IMAGE} -o ./bin/manager/manager.tar.gz
```

---

Good catch, done.